Source/WebKit/Platform/spi/ios/BaseBoardSPI.h

```diff
@@ -33,4 +33,30 @@
 - (void)invalidate;
 @end
 
+@interface BSSettings : NSObject
```

Comment:
In the `#if USE(APPLE_INTERNAL_SDK)` section, it seems like we might be missing some imports?

Reply:
Yes. Whatever they are, they arrive in the compilation units indirectly through some other imports.